src/Models/Site.php

```diff
@@ -86,6 +91,31 @@ public function exceptionLogGroups(): HasMany
         return $this->hasMany(ExceptionLogGroupRepository::resolveModelClass());
     }
 
+    public function systemMetrics(): HasMany
+    {
+        return $this->hasMany(SystemMetric::class);
+    }
+
+    public function getCpuLimit(): int
+    {
+        return $this->cpu_limit ?? 0;
+    }
+
+    public function getRamLimit(): int
+    {
+        return $this->ram_limit ?? 0;
+    }
+
+    public function getDiskLimit(): int
+    {
+        return $this->disk_limit ?? 0;
+    }
+
+    public function getMonitoringNotificationEnabled(): bool
+    {
+        return $this->monitoring_notification_enabled;
+    }
```

Comment:
This can be removed, the property is already casted into boolean. It should be used with:

$site->monitoring_notification_enabled